dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -10,8 +10,11 @@
 # See http://support.applitools.com/customer/en/portal/articles/2099488-match-timeout
 MATCH_TIMEOUT = 5
 
-When(/^I open my eyes to test "([^"]*)"$/) do |test_name|
-  next if CDO.disable_all_eyes_running
+When(/^I open my eyes to test "([^"]*)"( in test environment only)?$/) do |test_name, test_environment_only|
+  # Temporarily disable Eyes in CircleCI environments for tests that specify " in test environment only" because
+  # they generate different checkpoint images in the dedicated test environment vs in CircleCI builds.
+  # TODO: (suresh) remove the optional environment argument when there are no more tests using this option.
+  next if CDO.disable_all_eyes_running || (test_environment_only && ENV['IS_CIRCLE'] == 'true')
```

Comment:
This wording doesn't make sense to me because both the test machine and CircleCI run with rails and rack environments equal to "test". Furthermore you are not checking whether this happens on the test machine, rather you are checking that it does not happen in Circle. I would recommend changing `test_environment_only` to `except_in_circle`. Alternately, you could change it to `test_machine_only` and then change your logic to check explicitly that we are on the test machine instead.

Reply:
`except_in_circle` is much more clear.  I've implemented and tested that change.